flow-typed/box-ui-elements.js

```diff
@@ -711,7 +711,9 @@ type TaskLinkCollection = MarkerPaginatedCollection<TaskLink>;
 // See https://github.com/facebook/flow/issues/7574
 // This is currently *not* enforcing the constant types
 // type TaskType = typeof TASK_TYPE_GENERAL | typeof TASK_TYPE_APPROVAL;
+// type TaskEditMode = typeof TASK_EDIT_MODE_CREATE | typeof TASK_EDIT_MODE_EDIT;
```

Comment:
Are these to mark replacements once the flow issue is fixed?

Reply:
Correct. I haven't investigated that issue myself, but wanted to add a matching template so no one would have to guess what to do.